client/state/themes/selectors.js

```diff
@@ -741,3 +742,22 @@ export function getPremiumThemePrice( state, themeId, siteId ) {
 	const theme = getTheme( state, 'wpcom', themeId );
 	return get( theme, 'price' );
 }
+
+/**
+ * Checks if a theme should be customized primarily with Gutenberg.
+ *
+ * Examples include Template First Themes, which can be determined by the feature
+ * global-styles or auto-loading-homepage.
+ *
+ * @param {Object} state   Global state tree
+ * @param {String} themeId An identifier for the theme - like
+ *                         `independent-publisher-2` or `maywood`.
+ * @return {Boolean} True if the theme should be edited with gutenberg.
+ */
+export function shouldEditThemeWithGutenberg( state, themeId ) {
+	const theme = getTheme( state, 'wpcom', themeId );
+	const themeFeatures = getThemeTaxonomySlugs( theme, 'theme_feature' );
+	const neededFeatures = [ 'global-styles', 'auto-loading-homepage' ];
+	// The theme should have a positive number of matching features to qualify.
+	return !! intersection( themeFeatures, neededFeatures ).length;
+}
```

Comment:
I'm a bit rusty with Calypso conventions these days, but I think we should still keep adding new selectors in the `state/selectors` folder. 🤔

Reply:
this is where all of the theme-related selectors exist, and some other similar ones had been added here so I figured it'd be best to keep it consistent with that